api/client/proto/authservice.proto

```diff
@@ -373,6 +373,8 @@ message PingResponse {
     Features ServerFeatures = 3 [ (gogoproto.jsontag) = "server_features" ];
     // ProxyPublicAddr is the server's public proxy address.
     string ProxyPublicAddr = 4 [ (gogoproto.jsontag) = "proxy_public_addr" ];
+    // IsBoring signals whether or not the server was compiled with BoringCrypto.
+    bool IsBoring = 5 [ (gogoproto.jsontag) = "is_boring" ];
```

Comment:
This is not exposed on the public unauth `/ping` endpoint right?

Reply:
Correct. This is only exposed on auth's grpc server.